batch/batch/worker.py

```diff
@@ -732,7 +732,7 @@ def __init__(self):
 
             idle_duration = time_msecs() - self.last_updated
             while self.jobs or idle_duration < MAX_IDLE_TIME_MSECS:
-                log.info(f'n_jobs {len(self.jobs)} free_cores {self.free_cores_mcpu / 1000} idle {idle_duration}')
+                log.info(f'n_jobs {len(self.jobs)} free_cores {self.cpu_sem.value / 1000} idle {idle_duration}')
```

Comment:
Oops, one more thing: I think you can delete the assignment (now unused) to free_cores_mcpu.